.github/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 ### Improvements
 
+* Use `Operator.name` instead of `Operation.base_name`.
```

Comment:
```suggestion
* Use `Operator.name` instead of `Operation.base_name`.
  [(#115)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/115)
```